src/processes/command/subexports.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs'
+import path from 'path'
+import yargs from 'yargs'
+import fse from 'fs-extra'
+import { loadConfig } from '@/utils'
+
+const commandHandler = () => {
+    const args = loadConfig('subexports')
+    const root = fse.readJsonSync('./package.json').name
+
+    Object.entries(args)
+        .filter(([pkg, options]) => {
+            const isExists = fs.existsSync(path.resolve(process.cwd(), options.dist))
+            !isExists && console.error(`could not find dist for ${pkg} package`)
+            return isExists
+        })
+        .forEach(([pkg, options]) => {
+            const pkgFolder = path.resolve(process.cwd(), pkg)
+            const pkgSource = path.relative(pkgFolder, options.dist)
+
+            const pkgExport = {
+                private: true,
+                main: pkgSource,
```

Comment:
We need to support CJS also. The best would be if we would simply specify standard `package.json` fields, specifying `main` only automatically add it to `module` and `jsnext:main`.

e.g.

```
{
  subexports: {
     foo: { main: './dist/index.js' }
  } 
}
```